quazip/quaziodevice.cpp

```diff
@@ -155,13 +155,13 @@ bool QuaZIODevice::open(QIODevice::OpenMode mode)
     }
     if ((mode & QIODevice::ReadOnly) != 0) {
         if (inflateInit(&d->zins) != Z_OK) {
-            setErrorString(d->zins.msg);
+            setErrorString(QString::fromStdString(d->zins.msg));
```

Comment:
This call is suboptimal. It creates a temporary `std::string`, which is totally unnecessary. Besides, starting with Qt 5 it uses UTF-8, and I doubt that zlib always uses UTF-8. Please change this and similar calls to `QString::fromLocal8Bit`.